parlai/agents/bert_ranker/bert_dictionary.py

```diff
@@ -34,9 +34,9 @@ def __init__(self, opt):
         self.end_idx = self.tokenizer.convert_tokens_to_ids(['[SEP]'])[
             0]  # should be 102
         self.pad_idx = self.tokenizer.convert_tokens_to_ids(['[PAD]'])[0]  # should be 0
-        self[self.start_token] = self.start_idx
-        self[self.end_token] = self.end_idx
-        self[self.null_token] = self.pad_idx
+        self.tok2ind[self.start_token] = self.start_idx
+        self.tok2ind[self.end_token] = self.end_idx
+        self.tok2ind[self.null_token] = self.pad_idx
```

Comment:
I think you might also need to change self.ind2tok, no?

Reply:
technically no, since we only ever make calls to "vec2txt" which invokes the bert tokenizer and not the actual dictionary

Reply:
this fix just allows TorchAgent to set `self.START_IDX` etc properly

Reply:
yeah but... for completeness at least?